roles/tpa_single_node/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Get the Host IP
```

Comment:
That's obvious now that I see it, good idea  !

Reply:
Task removed based on our discussion on the meeting.